src/crypto_wrappers/openssl/gen_cert.c

```diff
@@ -102,8 +102,8 @@ crypto_wrapper_err_t openssl_gen_cert(crypto_wrapper_ctx_t *ctx, rats_tls_cert_a
 
 	X509_set_version(cert, 3);
 	ASN1_INTEGER_set(X509_get_serialNumber(cert), CERT_SERIAL_NUMBER);
-	/* 0 indicate start from the current time */
-	X509_gmtime_adj(X509_get_notBefore(cert), 0);
+	/* WORKAROUND: allow 1 hour delay for the systems behind current clock */
+	X509_gmtime_adj(X509_get_notBefore(cert), -3600);
 	/* 10 years */
```

Comment:
My question is why choose "1 hour" as the reference value instead of other value? Is there any convention for such value?

Reply:
There is no any convention here. Just a intuition.

Reply:
ok